cnd/src/swap_protocols/facade.rs

```diff
@@ -34,6 +38,96 @@ pub struct Herc20HalightBitcoinCreateSwapParams {
     pub lightning_amount: asset::Bitcoin,
 }
 
+/// This represent the information available on a swap
```

Comment:
🔧 
```suggestion
/// This represents the information available on a swap
```

Reply:
`Herc20HbitSwapParams` below is missing a similar docstring.

Reply:
Lol, I egotistically did not fix this because I don't like these structs :)  Will fix.